invenio_rdm_records/resources/config.py

```diff
@@ -122,3 +122,26 @@ class RDMManagedPIDProviderResourceConfig(RecordResourceConfig):
     response_handlers = {
         "application/json": ResponseHandler(JSONSerializer())
     }
+
+
+class PIDResolverResourceConfig(RecordResourceConfig):
+    """PID provider resource configuration."""
+
+    blueprint_name = "record_pids_resolver"
+
+    url_prefix = "/pids/<pid_type>"
+
+    routes = {
+        "item-doi": "/<pid_prefix>/<pid_value>",
+        "item": "/<pid_value>"
```

Comment:
You'll have the same issue in Invenio-App-RDM, so instead of using two endpoints, simply use one:

```
"item": "/<path:pid_value>"
```

Reply:
We discussed about this, the issue is the `/` in the doi value, the idea was to avoid to try to split the value by `/` manually.